packages/services/src/__tests__/rewards.service.test.ts

```diff
@@ -258,217 +211,20 @@ describe("RewardsService", () => {
-
-  describe("retrieveProof", () => {
-    const competitionId = "comp-123";
-    const address =
-      "0x1234567890123456789012345678901234567890" as `0x${string}`;
-    const amount = 100n;
-
-    it("should successfully retrieve and validate proof for valid reward", async () => {
-      const leafHash = createLeafNode(address, amount);
-      const siblingHash = new Uint8Array([1, 2, 3]);
-
-      // Convert hex string to Uint8Array for proper comparison
-      const leafHashBytes = new Uint8Array(
-        Buffer.from(leafHash.slice(2), "hex"),
-      );
-
-      // Calculate parent hash using proper byte arrays
-      const parentHashBytes = keccak256(
-        Buffer.concat(
-          [Buffer.from(siblingHash), Buffer.from(leafHashBytes)].sort(
-            Buffer.compare,
-          ),
-        ),
-      );
-
-      // Create a realistic tree structure where we can validate the proof
-      const mockTree: SelectRewardsTree[] = [
-        {
-          id: "node-1",
-          competitionId,
-          level: 0,
-          idx: 0,
-          hash: siblingHash, // sibling of our target leaf
-          createdAt: new Date(),
-        },
-        {
-          id: "node-2",
-          competitionId,
-          level: 0,
-          idx: 1,
-          hash: leafHashBytes, // our target leaf at position 1
-          createdAt: new Date(),
-        },
-        {
-          id: "node-3",
-          competitionId,
-          level: 1,
-          idx: 0,
-          hash: new Uint8Array(Buffer.from(parentHashBytes.slice(2), "hex")),
-          createdAt: new Date(),
-        },
-      ];
-
-      vi.mocked(
-        rewardsRepository.getRewardsTreeByCompetition,
-      ).mockResolvedValue(mockTree);
-
-      const service = new RewardsService();
-      const proof = await service.retrieveProof(competitionId, address, amount);
-
-      expect(proof).toBeInstanceOf(Array);
-      expect(proof.length).toBe(1); // Should have one sibling proof
-      expect(proof[0]).toEqual(siblingHash); // Should contain the sibling hash
-
-      // Verify proof can reconstruct to the parent node
-      const reconstructedParent = keccak256(
-        Buffer.concat(
-          [Buffer.from(siblingHash), Buffer.from(leafHashBytes)].sort(
-            Buffer.compare,
-          ),
-        ),
-      );
-      expect(
-        new Uint8Array(Buffer.from(reconstructedParent.slice(2), "hex")),
-      ).toEqual(mockTree[2]?.hash);
-    });
-
-    it("should throw error when reward not found in tree", async () => {
-      const mockTree: SelectRewardsTree[] = [
-        {
-          id: "node-1",
-          competitionId,
-          level: 0,
-          idx: 0,
-          hash: new Uint8Array([1, 2, 3]),
-          createdAt: new Date(),
-        },
-      ];
-
-      vi.mocked(
-        rewardsRepository.getRewardsTreeByCompetition,
-      ).mockResolvedValue(mockTree);
-
-      const service = new RewardsService();
+      ).rejects.toThrow("Blockchain allocation failed");
```

Comment:
blegh the new github code review tool isn't letting me comment on deleted lines, but it looks like a bunch of test cases in this file got deleted - was that intentional?

Reply:
lol I guess those comments actually did get recorded the UI just wouldn't show them to me 😅 

How is the github code review tool actually getting worse?!